crates/primitives/block/src/header.rs

```diff
@@ -159,11 +159,38 @@ impl Header {
                 self.parent_block_hash.into(),
             ];
 
-            // for (i, item) in data.iter().enumerate() {
-            //     frame_support::log::info!("data[{}]: {:?}", i, item);
-            // }
-
             H::compute_hash_on_wrappers(data)
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use mp_hashers::pedersen::PedersenHasher;
+
+    use super::*;
+
+    #[test]
+    fn test_header_hash() {
```

Comment:
Please consider documenting this test and it's purpose

Reply:
removed as it is useless for now